way-cooler/config/init.lua

```diff
@@ -114,3 +114,9 @@ end
 
 -- !! Do not place any code after this comment.
 -- !! way-cooler and plugins may insert auto-generated code.
+local status = os.execute("which way-cooler-bg 2>/dev/null")
```

Comment:
Yes, that's what I wanted to do, but didn't have enough Lua/hlua knowledge to know how to go about it (in particular, the thing I was planning was for a Lua function to be registered to be called after wlc init, which could then spawn the program, and provide a nice dumping ground for other "startup" programs). 

If you wouldn't mind, it would be much faster if you made that change.